roles/internal/Islandora-Devops.alpaca/templates/startup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ -f "/usr/libexec/tomcat9/tomcat-locate-java.sh" ]; then
+  . /usr/libexec/tomcat9/tomcat-locate-java.sh
+fi
```

Comment:
Yeah, I didn't want to copy the code and `$JAVA_HOME` doesn't seem to be set by default. Because this is only for Ubuntu I'll just assume where java is.

Reply:
Made a guess based on where Ubuntu should put OpenJDK 11. Resolved in https://github.com/Islandora-Devops/islandora-playbook/pull/208/commits/a7414625efea5868dd0d22598cee559e7928003c